renderer.js

```diff
@@ -26,3 +28,29 @@ const validateUrl = (link) => {
   }
   return link;
 }
+
+// Helper function, to format the time
+const secondsToTime = (s) => {
+  let momentTime = moment.duration(s, 'seconds');
+  let sec = momentTime.seconds() < 10 ? ('0' + momentTime.seconds()) : momentTime.seconds();
+  let min = momentTime.minutes() < 10 ? ('0' + momentTime.minutes()) : momentTime.minutes();
+  return `${min}:${sec}`;
+}
+
+  // Initial time, hardcoded
+  let currentTime = 255;
+
+  // Print out the time
+  timerDiv.innerHTML = secondsToTime(currentTime);
+
+  // Execute every second
+  let timer = setInterval(() => {
```

Comment:
I was comparing this code to the code in the Wes Bos tutorial which used some more 'rithmetic:
`countdown = setInterval(() => {
    const secondsLeft = Math.round((then - Date.now()) / 1000);
    // check if we should stop it!
    if(secondsLeft < 0) {
      clearInterval(countdown);
      return;
    }
    // display it
    displayTimeLeft(secondsLeft);
  }, 1000);`